int32 from_drone_id
float64 send_time

DivisionData[] divs